requirements/development.txt

```diff
@@ -24,3 +24,4 @@ numpy
 scikit-allel
 zarr>=2.4
 biopython
+boto3
```

Comment:
What's this for? I didn't see it in the imports.

Reply:
ah thats for the AWS push script I wrote. still not sure how to share that script cause of passwords.

Reply:
this is for the `aws_push_annotations.py` script that i've written to deal with getting these info to AWS. problem that we haven't yet figured out is how to share those passwords securely and have the code on github